src/kernel/src/dev/dmem.rs

```diff
@@ -71,7 +71,7 @@ impl DeviceDriver for Dmem {
             // TODO: properly implement this
             IoCmd::DMEMTOTAL(size) => *size = self.total_size,
             IoCmd::DMEMGETPRT(_prt) => todo!(),
-            IoCmd::DMEMGETAVAIL(_avail) => todo!(),
+            IoCmd::DMEMGETAVAIL(_avail) => {}
```

Comment:
Why remove `todo` here?

Reply:
I put it back

Reply:
It seems to work without it, but we probably shouldn't play around with fire.